src/gui/transferlistmodel.cpp

```diff
@@ -357,6 +358,13 @@ QString TransferListModel::displayValue(const BitTorrent::Torrent *torrent, cons
         return Utils::Misc::userFriendlyDuration(time);
     };
 
+    const auto privateString = [](const bool isPrivate) -> QString
+    {
+        if (isPrivate)
+            return QStringLiteral("Yes");
+        return QStringLiteral("No");
```

Comment:
```suggestion
        return tr("No");
```

Same in all other places where you use `QStringLiteral`

Reply:
Done